src/PointwiseFunctions/GeneralRelativity/ComputeGhQuantities.cpp

```diff
@@ -196,21 +196,32 @@ tnsr::a<DataType, SpatialDim, Frame> gauge_source(
 }
 
 template <size_t SpatialDim, typename Frame, typename DataType>
-tnsr::ii<DataType, SpatialDim, Frame> extrinsic_curvature(
+void extrinsic_curvature(
+    const gsl::not_null<tnsr::ii<DataType, SpatialDim, Frame>*> ex_curv,
     const tnsr::A<DataType, SpatialDim, Frame>& spacetime_normal_vector,
     const tnsr::aa<DataType, SpatialDim, Frame>& pi,
     const tnsr::iaa<DataType, SpatialDim, Frame>& phi) noexcept {
-  auto ex_curv = make_with_value<tnsr::ii<DataType, SpatialDim, Frame>>(pi, 0.);
+  *ex_curv = make_with_value<tnsr::ii<DataType, SpatialDim, Frame>>(pi, 0.);
```

Comment:
would `destructive_resize` work here? This current does a bunch of allocations even if the tensor is already allocated